source/validate.cpp

```diff
@@ -286,6 +291,12 @@ spv_result_t spvValidateIDs(const spv_instruction_t* pInsts,
 }
 
 namespace {
+// Shame
```

Comment:
Clang complains about using assignment in a condition statement. I could have added `#pragma clang diagnostic ignore -Wparentheses` but that seemed uglier at the moment.